sdk/cosmosdb/Microsoft.Azure.Management.CosmosDB/src/Microsoft.Azure.Management.CosmosDB.csproj

```diff
@@ -6,17 +6,17 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.CosmosDB</PackageId>
     <Description>Provides developers with libraries for the CosmosDB under Azure Resource manager to manage CosmosDB Account, Databases and child resources and their available management capabilities. Create, Delete, Update CosmosDB Account, Databases and child resources. Note: This client library is for CosmosDB under Azure Resource Manager.</Description>
-    <Version>1.2.0</Version>
+    <Version>1.3.0-preview</Version>
     <AssemblyName>Microsoft.Azure.Management.CosmosDB</AssemblyName>
     <PackageTags>management;cosmosdb;</PackageTags>
     <PackageReleaseNotes>
       <![CDATA[
-        Updating with the apiVersion 2020-04-01 azure-rest-api-specs,
+        Updating with the apiVersion 2020-06-01-preview azure-rest-api-specs,
         Includes: 
```

Comment:
Can you elaborate  the changes a little more? Explain what is included in DatabaseAccountCreateUpdateProperties. Sum all the changes related to PITR in one para. Please mention, this is for the private preview of PITR.

Reply:
Elaborated the changes now, Please let me know if you would like me to expand more.